### ISSUE TITLE
Unsupported Unicode escape sequence causing fail 

### ISSUE THREAD

**Issue by MaxwellDPS** [intelowlproject/IntelOwl#138]
This is the error below, Censys causes one via IP.. Ubuntu 20.04 

>Linux intelowl 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

```
[
  "unsupported Unicode escape sequence\nLINE 1: UPDATE \"api_app_job\" SET \"analysis_reports\" = '[{\"name\": \"Cy...\n                                                      ^\nDETAIL:  \\u0000 cannot be converted to text.\nCONTEXT:  JSON data, line 1: ...: \"Mimikatz_Strings\", \"strings\": [[770880, \"$x2\",...\n"
]
```

**Comment by mlodic** [comment]
Thank you for your report! 

This is a known problem that could happen with each analyzer, depending on the data that the analyzer gives back to the user. This is the issue explained: https://stackoverflow.com/questions/31671634/handling-unicode-sequences-in-postgresql

We should create a sanitizer at the database level to avoid this problem. We will update this issue once we have news about on how to solve it. Every suggestions is welcome.

**Comment by mlodic** [comment]
fixed here #223 and available in the next release
